src/reflect/type.go

```diff
@@ -395,8 +395,10 @@ type Type interface {
 
 // Constants for the 'meta' byte.
 const (
-	kindMask  = 31 // mask to apply to the meta byte to get the Kind value
-	flagNamed = 32 // flag that is set if this is a named type
+	kindMask       = 31  // mask to apply to the meta byte to get the Kind value
+	flagNamed      = 32  // flag that is set if this is a named type
+	flagComparable = 64  // flag that is set if this type is comparable
+	flagIsBinary   = 128 // flag that is set if this type uses the hashmap binary algorithm
```

Comment:
Perhaps `1 << 5` etc would have been more readable.
I'm fine with it either way.